pkgs/applications/editors/vim/plugins/update.py

```diff
@@ -1,181 +1,238 @@
-#!/usr/bin/env python
-
-# run with:
-# $ nix run .\#vimPluginsUpdater
-# format:
-# $ nix run nixpkgs#python3Packages.ruff -- update.py
-# type-check:
-# $ nix run nixpkgs#python3Packages.mypy -- update.py
-# linted:
-# $ nix run nixpkgs#python3Packages.flake8 -- --ignore E501,E265,E402 update.py
-
-# If you see `HTTP Error 429: too many requests` errors while running this
-# script, refer to:
-#
-# https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/vim.section.md#updating-plugins-in-nixpkgs-updating-plugins-in-nixpkgs
-#
-# (or the equivalent file /doc/languages-frameworks/vim.section.md
-# from Nixpkgs master tree).
-#
```

Comment:
We probably still want some "run with" examples in this file, maybe also a link to the relevant docs.

Given it has a nix-shell shebang, I'm guessing it'll be something like "run with ./update.py"?